plugins/Goals/tests/System/expected/test_trackGoals_pagesshowGoalsMetricsSingleGoal__API.getProcessedReport_day.xml

```diff
@@ -104,44 +104,19 @@
 		</row>
 	</reportMetadata>
 	<reportTotal>
-		<nb_visits>4</nb_visits>
-		<nb_uniq_visitors>4</nb_uniq_visitors>
-		<nb_hits>5</nb_hits>
-		<sum_time_spent>1800</sum_time_spent>
-		<sum_bandwidth>0</sum_bandwidth>
-		<nb_hits_with_bandwidth>0</nb_hits_with_bandwidth>
-		<entry_nb_uniq_visitors>4</entry_nb_uniq_visitors>
 		<entry_nb_visits>4</entry_nb_visits>
-		<entry_nb_actions>18</entry_nb_actions>
-		<entry_sum_visit_length>5047</entry_sum_visit_length>
 		<entry_bounce_count>0</entry_bounce_count>
-		<goals>
-			<row idgoal="1">
-				<nb_conversions>3</nb_conversions>
-				<revenue>30</revenue>
-				<nb_conv_pages_before>4</nb_conv_pages_before>
-				<nb_conversions_attrib>1.0833</nb_conversions_attrib>
-				<nb_conversions_page_uniq>3</nb_conversions_page_uniq>
-				<revenue_attrib>10.83</revenue_attrib>
-				<revenue_entry>30</revenue_entry>
-				<revenue_per_entry>7.5</revenue_per_entry>
-				<nb_conversions_entry>3</nb_conversions_entry>
-				<nb_conversions_page_rate>1</nb_conversions_page_rate>
-				<nb_conversions_entry_rate>0.75</nb_conversions_entry_rate>
-			</row>
-			<row idgoal="2">
-				<nb_conversions>2</nb_conversions>
-				<revenue>20</revenue>
-				<nb_conv_pages_before>5</nb_conv_pages_before>
-				<nb_conversions_attrib>0.4</nb_conversions_attrib>
-				<nb_conversions_page_uniq>2</nb_conversions_page_uniq>
-				<revenue_attrib>4</revenue_attrib>
-				<revenue_entry>10</revenue_entry>
-				<revenue_per_entry>2.5</revenue_per_entry>
-				<nb_conversions_entry>1</nb_conversions_entry>
-				<nb_conversions_page_rate>1</nb_conversions_page_rate>
-				<nb_conversions_entry_rate>0.25</nb_conversions_entry_rate>
-			</row>
-		</goals>
+		<goal_1_nb_conversions>3</goal_1_nb_conversions>
+		<goal_1_revenue_per_visit>7.5</goal_1_revenue_per_visit>
+		<goal_1_revenue>30</goal_1_revenue>
+		<goal_1_nb_conversions_entry>3</goal_1_nb_conversions_entry>
+		<goal_1_revenue_per_entry>7.5</goal_1_revenue_per_entry>
+		<goal_1_revenue_entry>30</goal_1_revenue_entry>
+		<goal_2_nb_conversions>2</goal_2_nb_conversions>
+		<goal_2_revenue_per_visit>5</goal_2_revenue_per_visit>
+		<goal_2_revenue>20</goal_2_revenue>
+		<goal_2_nb_conversions_entry>1</goal_2_nb_conversions_entry>
+		<goal_2_revenue_per_entry>5</goal_2_revenue_per_entry>
+		<goal_2_revenue_entry>10</goal_2_revenue_entry>
```

Comment:
Just in case someone wonders why the goal metrics `conversion_rate` and `nb_conversions_entry_rate` are missing here. This is because they are removed from processed report totals here:

https://github.com/matomo-org/matomo/blob/4a2025e80eff927f7dc9a29f8953119e6080d437/plugins/API/ProcessedReport.php#L808-L810

It should though be possible to calculate them using the processed metric class somehow, but that might imply some bigger refactoring. So out of scope here.